templates/forms/fields/textarea/textarea.html.twig

```diff
@@ -1,7 +1,19 @@
 {% extends "forms/field.html.twig" %}
 
+{% block label %}
+{% endblock %}
+
 {% block input %}
+    {% set id = field.id|default(field.name) ~ '-' ~ key %}
     <div class="{{ form_field_wrapper_classes ?: 'form-textarea-wrapper' }} {{ field.size }} {{ field.wrapper_classes }}">
+        <label style="display:inline;" class="inline" for="{{ id|e }}">
```

Comment:
What is the purpose of putting the label in this input block and not the label block? This will change the display on all forms with textarea as the div/class structure is changed.

Reply:
Yes, that's right. I've rearranged the code now. Let me know if that looks better!